libqtile/widget/wttr.py

```diff
@@ -67,7 +67,7 @@ class Wttr(GenPollUrl):
         (
             'units', 'm',
             "``'m'`` - metric, ``'M'`` - show wind speed in m/s, "
-            "``'s'`` - imperial"
+            "``'u'`` - USCS"
```

Comment:
I don't know how familiar people are with the `USCS` acronym (I've not heard it before). Perhaps change it to "United States units".

Reply:
Last request: Can you squash those into a single commit and force push it? I can then take it from there.

Appreciate you doing this given you're not a Qtile user.